json_rpc/rpcproxy.nim

```diff
@@ -0,0 +1,38 @@
+import servers/[socketserver, httpserver]
+import clients/[socketclient, httpclient]
+
+type RpcHttpProxy* = ref object of RootRef
+    rpcHttpClient: RpcHttpClient
+    rpcHttpServer: RpcHttpServer
+
+proc proxyCall(client: RpcHttpClient): ProxyCall =
+  result = 
```

Comment:
For new code we avoid using result: https://status-im.github.io/nim-style-guide/language.result.html